src/sdk.ts

```
@@ -39,6 +40,7 @@ import type {
   ZipBundleOptions,
 } from './schemas/sdk'
 import type { Organization } from './utils'
+import type { ProbeInternalResult } from './probe'
```

ESLint

[error] Expected "./probe" to come before "./utils".

(perfectionist/sort-imports)